src/neural/network_opencl.cc

```diff
@@ -152,10 +132,9 @@ class OpenCLNetwork : public Network {
       : weights_(weights), params_(), opencl_(), opencl_net_(opencl_) {
     params_.gpuId = options.GetOrDefault<int>("gpu", -1);
     params_.verbose = options.GetOrDefault<bool>("verbose", false);
-    params_.force_tune = options.GetOrDefault<int>("force_tune", false);
-    params_.tune_only = options.GetOrDefault<int>("tune_only", false);
-    params_.tune_exhaustive =
-        options.GetOrDefault<int>("tune_exhaustive", false);
+    params_.force_tune = options.GetOrDefault<bool>("force_tune", false);
```

Comment:
Alternatively, 
`options.GetOrDefault<string>("tune", "auto")`, with acceptable values "force", "only", "auto" and "exhaustive".

But I'm not convinced that it would be better.

Reply:
Not too convinced too. You could want to force an exhaustive tune, or force a non-exhaustive tune. Almost all 2^3 combinations make sense here.